tools/ratbagctl.test.in

```diff
@@ -336,6 +342,22 @@ class TestRatbagCtlResolution(TestRatbagCtl):
         r = self.launch_good_test("test_device profile 2 resolution 1 get")
         self.assertEqual(r, "1: 1200x1300dpi")
 
+    def test_resolution_disabled_get(self):
+        command = "resolution 3 disabled get"
+        r = self.launch_good_test("test_device " + command)
+        self.assertEqual(r, "False")
+        self.launch_fail_test("test_device " + command + " X")
+        self.launch_fail_test("test_device " + command + " 10")
+
+    def test_resolution_disabled_set(self):
+        command = "resolution 3 disabled set"
+        self.launch_good_test("test_device " + command)
+        r = self.launch_good_test("test_device resolution 3 disabled get")
+        self.assertEqual(r, "True")
+        self.launch_fail_test(command)
+        self.launch_fail_test("test_device " + command + " X")
+        self.launch_fail_test("test_device " + command + " 1 X")
```

Comment:
What exactly are these `launch_fail_test`'s? It works but I'm not sure if they even make sense here.